consensus/src/quorum_store/types.rs

```diff
@@ -58,14 +58,15 @@ impl PersistedValue {
         &self.maybe_payload
     }
 
-    pub fn summary(&self) -> Vec<TransactionSummary> {
+    pub fn synopsis(&self) -> Vec<TxnSummaryWithExpiration> {
```

Comment:
Why is this renamed to `synopsis`?

Reply:
I actually used TransactionSynopsis earlier. After Sital's comment, I renamed it to TxnSummaryWithExpiration and forgot to update this statement. Updated it now.